creator-node/src/ipfsAdd.js

```diff
@@ -99,147 +96,48 @@ ipfsAdd.ipfsOnlyHashImages = async (content, options = {}) => {
 }
 
 /**
- * Wrapper for ipfs.add() with flag to exclusively generate the CID using the only hash logic, or to also add content to the ipfs daemon.
- * Used for adding non-images (track segments, track transcode, metadata).
- * @param {Object} ipfsLatest ipfs instance (should be v43.0.1)
+ * Wrapper that generates multihashes for non-image files (track segments, track transcode, metadata)
  * @param {Buffer|ReadStream|string} content a single Buffer, a ReadStream, or path to an existing file
- * @param {Object?} ipfsConfig ipfs add config options
  * @param {Object?} logContext
- * @param {boolean?} enableIPFSAdd flag to add content to ipfs daemon
  * @returns {string} only hash response cid or ipfs daemon response cid
  */
-ipfsAdd.ipfsAddNonImages = async (
-  content,
-  ipfsConfig = {},
-  logContext = {},
-  enableIPFSAdd = false
-) => {
+ipfsAdd.generateNonImageMultihashes = async (content, logContext = {}) => {
```

Comment:
this fn should only return 1 multihash. let's rename this to `generateNonImageMultihash`

Reply:
good point -- changing